hack/update-vault.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+sed -i -z "s/kubectl create ns vault/oc delete project vault --ignore-not-found=true\n  kubectl create ns vault\n  kubectl label ns vault security.openshift.io\/scc.podSecurityLabelSync=false pod-security.kubernetes.io\/enforce=privileged pod-security.kubernetes.io\/audit=privileged pod-security.kubernetes.io\/warn=privileged --overwrite\n  oc adm policy add-scc-to-user privileged system:serviceaccount:vault:vault\n  oc adm policy add-scc-to-user privileged system:serviceaccount:vault:vault-csi-provider/g" `pwd`/test/bats/vault.bats
+sed -i -z "s/--set \"csi.enabled=true\"/--set \"csi.enabled=true\" \\\\\n        --set \"global.openshift=true\" \\\\\n        --set \"csi.daemonSet.providersDir=\/var\/run\/secrets-store-csi-providers\"\n  oc patch daemonset -n vault vault-csi-provider --type='json' -p='[{\"op\": \"add\", \"path\": \"\/spec\/template\/spec\/containers\/0\/securityContext\", \"value\": {\"privileged\": true} }]'\n  sleep 10/g" `pwd`/test/bats/vault.bats
+
+sed -i -z "s/kubectl create ns test-ns/oc delete project test-ns --ignore-not-found=true\n  kubectl create ns test-ns\n  kubectl label ns test-ns security.openshift.io\/scc.podSecurityLabelSync=false pod-security.kubernetes.io\/enforce=privileged pod-security.kubernetes.io\/audit=privileged pod-security.kubernetes.io\/warn=privileged --overwrite/g" `pwd`/test/bats/vault.bats
+sed -i -z "s/kubectl create ns negative-test-ns/oc delete project negative-test-ns --ignore-not-found=true\n  kubectl create ns negative-test-ns\n  kubectl label ns negative-test-ns security.openshift.io\/scc.podSecurityLabelSync=false pod-security.kubernetes.io\/enforce=privileged pod-security.kubernetes.io\/audit=privileged pod-security.kubernetes.io\/warn=privileged --overwrite/g" `pwd`/test/bats/vault.bats
+
+sed -i -z "s/sleep 60/sleep 90/g" `pwd`/test/bats/vault.bats
```

Comment:
Is this mandatory? I see upstream cases passed always, do we hit any issues?

Reply:
yeah atleast wrt rotation tc 70% times failed with sleep 60 and with sleep 90 always passed on OCP. Might be time sync issue with 60sec or not sure.